bindings/node/index.d.ts

```diff
@@ -497,6 +533,21 @@ export class ClientEncryption {
    */
   removeKeyAltName(id: Binary, keyAltName: string): Promise<DataKey | null>;
 
+  /**
+   * Creates a collection that has encrypted document fields.
+   * Will create dataKeys for any fields missing a `keyId`
+   *
+   * @param db - The database to create the collection within
+   * @param name - The name of the new collection
+   * @param options - Options for createDataKey and for createCollection. A provider and partially created encryptedFields **must** be provided.
+   * @throws {MongoCryptCreateEncryptedCollectionError} - If part way through the process createDataKey fails, an error will be rejected that has the `encryptedFields` that were created.
+   */
+  createEncryptedCollection<TSchema extends Document = Document>(db: Db, name: string, options: {
+    provider: ClientEncryptionDataKeyProvider;
+    createCollectionOptions: Omit<CreateCollectionOptions, 'encryptedFields'> & { encryptedFields: NonNullable<CreateCollectionOptions['encryptedFields']> };
```

Comment:
We don't have types in the 3.x driver so I'm not worried. But this type would fail if `CreateCollectionOptions` does not have an `encryptedFields` property so should we leave this untyped / `Document`? it's frustrating that these could fall out of sync, but maybe it's safer.